frontend/src/pages/_app.js

```diff
@@ -55,7 +65,10 @@ function KitspaceApp({ Component, pageProps, session, isStaticFallback }) {
     isStaticFallback = isStaticFallback || window.isStaticFallback
   }
   return (
-    <AuthProvider>
+    <AuthProvider
+      initialCsrf={serverSideSession?.csrf || clientSideSession?.csrf}
+      initialUser={serverSideSession?.user || clientSideSession?.user}
+    >
```

Comment:
From the `AuthProvider` perspective, is there a difference between `serverSideSession`, and `clientSideSession`? I think `KitspaceApp` should get a single `session` Prop.

Reply:
From the `AuthProvider` perspective, no. But from the `KitspaceApp` perspective, they need to be handled very differently. That's why I gave them clearer names.

Reply:
We can keep it for now and get rid of it with `window.session`.